desk/ted/save-summary.hoon

```diff
@@ -45,6 +46,9 @@
       =;  vars=(map @t json)
         !>(`[mailchimp.args u.mail vars])
       %-  ~(gas by *(map @t json))
+      =?  summary.args  ?=(%wipe -.summary.args)
+        [%life [0 0 ''] [0 0 '' '']]
```

Comment:
Is there any difference between these values and uninitialized values in mailchimp?

Reply:
The strings are at least visually indistinguishable. Uninitialized values actually show up as empty, even the number fields, so those are distinguishable (0 vs empty).